test/functional/test-performance.js

```diff
@@ -380,35 +385,40 @@ describe('performance', () => {
           perf.tick('start0');
           perf.tick('start1', 'start0', 300);
 
-          expect(tickSpy.getCall(2).args[0]).to.be.jsonEqual({
-            label: 'start0',
-            from: null,
-            value: 100,
-          });
-          expect(tickSpy.getCall(3).args[0]).to.be.jsonEqual({
-            label: 'start1',
-            from: 'start0',
-            value: 300,
-          });
+          expect(viewerSendMessageStub.withArgs('tick').getCall(2).args[1])
+              .to.be.jsonEqual({
+                label: 'start0',
+                from: null,
+                value: 100,
+              });
+          expect(viewerSendMessageStub.withArgs('tick').getCall(3).args[1])
+              .to.be.jsonEqual({
+                label: 'start1',
+                from: 'start0',
+                value: 300,
+              });
         });
       });
 
       it('should call the flush callback', () => {
-        expect(flushTicksSpy.callCount).to.equal(0);
+        expect(viewerSendMessageCancelUnsentStub.withArgs('sendCsi')
+            .callCount).to.equal(0);
         // coreServicesAvailable calls flush once.
         return perf.coreServicesAvailable().then(() => {
-          expect(flushTicksSpy.callCount).to.equal(1);
+          expect(viewerSendMessageCancelUnsentStub.withArgs('sendCsi')
+              .callCount).to.equal(1);
```

Comment:
since we're in the process of making the test nicer too, can you try out the `spy.should.have.callCount(n)` syntax? https://github.com/domenic/sinon-chai